tests/openapi/runner/test_runner.py

```diff
@@ -0,0 +1,26 @@
+import os
+import unittest
+
+from checkov.openapi.runner import Runner
+from checkov.runner_filter import RunnerFilter
+
+class TestRunnerValid(unittest.TestCase):
+
+    def test_runner(self) -> None:
+        current_dir = os.path.dirname(os.path.realpath(__file__))
```

Comment:
```suggestion
        current_dir = os.path.dirname(__file__)
```
please try to not use `realpath` anywhere, if possible. it follows symbolic links, which results in weird behaviour.